content/logs/log_collection/docker.md

```diff
@@ -26,30 +26,30 @@ Two installations are possible:
 - on the host where the Agent is external to the Docker environment
 - or by deploying its containerized version in the Docker environment
 
-You can then choose to collect all the logs from all your environment container or to filter by container image name or container label to cherry pick what logs should be collected.
+You can then choose to collect all the logs from all your environment's containers or to filter by container image name or container label to cherry pick what logs should be collected.
 
 ## Setup
 ### Option 1: Host installation
 
 Install the [latest version of the Agent 6][1] on your host.
 
-The Agent can both collect logs from [files on the host][2] or from [container stdout/stderr](#configuration-file-example).
+The Agent can collect logs from [files on the host][2] or from [container stdout/stderr](#configuration-file-example).
 
-To collect logs from all your container without filtering by image or label add the following at the end of `docker.d/conf.yaml` in your agent's `conf.d` directory:
+To collect logs from all your containers without filtering by image or label add the following at the end of `docker.d/conf.yaml` in your agent's `conf.d` directory:
```

Comment:
To collect logs from all your containers without filtering by image or label, add the following at the end of `docker.d/conf.yaml` in your agent's `conf.d` directory: